bitnami/airflow/templates/worker-statefulsets.yaml

```diff
@@ -0,0 +1,309 @@
+        imagePullPolicy: {{ .image.pullPolicy default "IfNotPresent" }} 
+        {{- else }}
+        image: "docker.io/bitnami/airflow-worker:1.10.6-debian-9-r1"
+        imagePullPolicy: "IfNotPresent"
+        {{- end }}
+        resources:
+{{ toYaml $.Values.resources | indent 10 }}
+        env:
+          {{- if $.Values.workerImage.debug }}
+          - name: BASH_DEBUG
+            value: "1"
+          - name: NAMI_DEBUG
+            value: "1"
+          - name: NAMI_LOG_LEVEL
+            value: "trace8"
+          {{- end }}
+          - name: AIRFLOW_DATABASE_NAME
+            {{- if $.Values.postgresql.enabled }}
+            value: {{ $.Values.postgresql.postgresqlDatabase }}
+            {{- else }}
+            value: {{ $.Values.externalDatabase.database | quote }}
+            {{- end }}
+          - name: AIRFLOW_DATABASE_USERNAME
+            {{- if $.Values.postgresql.enabled }}
+            value: {{ $.Values.postgresql.postgresqlUsername }}
+            {{- else }}
+            value: {{ $.Values.externalDatabase.user | quote }}
+            {{- end }}
+          - name: AIRFLOW_DATABASE_PASSWORD
+            valueFrom:
+              secretKeyRef:
+              {{- if $.Values.postgresql.enabled }}
+                name: {{ template "airflow.postgresql.fullname" $ }}
+                key: postgresql-password
+              {{- else }}
+                name: {{ printf "%s-%s" .Release.Name "externaldb" }}
+                key: db-password
+              {{- end }}
+          - name: AIRFLOW_DATABASE_HOST
+            {{- if $.Values.postgresql.enabled }}
+            value: {{ template "airflow.postgresql.fullname" $ }}
+            {{- else }}
+            value: {{ $.Values.externalDatabase.host | quote }}
+            {{- end }}
+          - name: AIRFLOW_DATABASE_PORT_NUMBER
+            {{- if $.Values.postgresql.enabled }}
+            value: "5432"
+            {{- else }}
+            value: {{ $.Values.externalDatabase.port | quote }}
+            {{- end }}
+          - name: REDIS_HOST
+            {{- if $.Values.redis.enabled }}
+            value: {{ template "airflow.redis.fullname" $ }}
+            {{- else }}
+            value: {{ $.Values.externalRedis.host | quote }}
+            {{- end }}
+          - name: REDIS_PORT_NUMBER
+            {{- if $.Values.redis.enabled }}
+            value: "6379"
+            {{- else }}
+            value: {{ $.Values.externalRedis.port | quote }}
+            {{- end }}
+          - name: REDIS_PASSWORD
+            valueFrom:
+              secretKeyRef:
+              {{- if $.Values.redis.enabled }}
+                name: {{ template "airflow.redis.secretName" $ }}
+                key: redis-password
+              {{- else }}
+                name: {{ printf "%s-%s" .Release.Name "externalredis" }}
+                key: redis-password
+              {{- end }}
+          - name: AIRFLOW_EXECUTOR
```

Comment:
I presume that this value should be configurable, right? Different type of workers having different executors